analyses/mwi/dryspells_trigger/exploration/mwi_agg_exploration.Rmd

```diff
@@ -0,0 +1,382 @@
+---
+title: "Comparison of MWI Observational Aggregation Methods"
+author: "Seth Caldwell"
+date: "15/12/2021"
+output: ioslides_presentation
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+
+library(tidyverse)
+library(lubridate)
+library(glue)
+library(ggthemes)
+library(sf)
+library(stars)
+library(rasterVis)
+
+# set filepaths
+arc2_dir <- file.path(Sys.getenv("AA_DATA_DIR"), "public", "processed", "mwi", "arc2")
+cod_dir <- file.path(Sys.getenv("AA_DATA_DIR"), "public", "raw", "mwi", "cod_ab", "mwi_adm_nso_20181016_shp")
+
+# get methods dry spells data
+
+methods <- c("approximate_mask", "centroid", "touching")
+
+df_list <- map(methods, ~read_csv(
+  file.path(arc2_dir, glue("arc2_{.x}_mwi_dry_spells_14_days_32E_36E_20S_5S_master.csv"))
```

Comment:
getting an error that this file doesn't exist

Reply:
Didn't feel like debugging right now so didn't review the rest of the file yet..

Reply:
And for some reason it is stating that I don't have permission to open the html file. So let me know when you debugged this code or the html and I can review

Reply:
Are you still getting errors here @Tinkaa? I get none and the fine does exist on my machine.

Reply:
Ha, it was just because I'd tried to be more culturally sensitive and change master to main as Hannah suggested, forgot to change this!